test/integration/nxos.yaml

```diff
@@ -298,35 +298,45 @@
           name: nxos_logging
         when: "limit_to in ['*', 'nxos_logging']"
       rescue:
-        - set_fact: test_failed=true
```

Comment:
I'd say it's time to delete `nxos.yaml` and just use `ansible-test network-integration --inventory /path/to/inventory/for/engansiblecom nxos_.*`

Reply:
@gundalow We use `nxos.yaml` to run our CI tests here.  We have not used `ansible-test` previously so would want to make sure we understand the format of the command and any env variables if any to run the tests in this manner.  We need to update our CI if `nxos.yaml` is going away.  Thanks!